packages/ra-ui-materialui/src/layout/Menu.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+
+import { Resource, testDataProvider } from 'ra-core';
+import { defaultTheme, Admin } from 'react-admin';
+import { AppBar, Typography, Box } from '@mui/material';
+import { createTheme } from '@mui/material/styles';
+import { ToggleThemeButton } from '../button';
+import { Layout, Menu, SidebarToggleButton, Title } from '.';
+
+export default { title: 'ra-ui-materialui/layout/Menu' };
+
+const resources = ['Posts', 'Comments', 'Tags', 'Users', 'Orders', 'Reviews'];
+
+export const MenuDefault = () => (
+    <Admin dataProvider={testDataProvider()} layout={StorybookLayoutDefault}>
+        {resources.map((resource, index) => {
+            return (
+                <Resource
+                    name={resource}
+                    key={`ressource_${index}`}
+                    list={<StorybookList name={resource} />}
+                />
+            );
+        })}
+    </Admin>
+);
+const StorybookLayoutDefault = props => (
```

Comment:
Why the "Storybook" prefix? It seems useless. Same for the other components.

Reply:
I included single use slaves components within storybook cases components (Default, Dense) and left shared `DemoAppBar` and `DemoList` with a prefix to avoid conflicts.